.travis.yml

```diff
@@ -39,4 +39,5 @@ script:
   - set +e
 
 after_success:
+  - if [ "$TRAVIS_BRANCH" = "master" ] && [[ ! "$TRAVIS_COMMIT_MESSAGE" =~ \[skip\ publish\] ]]; then yarn release-canary; fi
```

Comment:
I tested this locally:

<img width="724" alt="image" src="https://user-images.githubusercontent.com/1110551/50635709-86d02180-0f53-11e9-9e88-ccd0bd6e7729.png">